lib/elixir/lib/keyword.ex

```diff
@@ -832,7 +832,7 @@ defmodule Keyword do
   """
   @spec drop(t, [key]) :: t
   def drop(keywords, keys) when is_list(keywords) do
-    :lists.filter(fn {k, _} -> not k in keys end, keywords)
+    :lists.filter(fn {k, _} -> k not in keys end, keywords)
```

Comment:
The ultimate goal is to replace single-letter variable throughout the file, ideally without making change commit, rather with time, every time when we touch the line we can replace single-letter variable. But probably we can ignore it this time, it's your call. :bowtie:
_Same applies to another place._